src/Rings/orderings.jl

```diff
@@ -629,6 +698,8 @@ function _cmp_monomials(f::MPolyRingElem, k::Int, g::MPolyRingElem, l::Int, o::S
   return 0
 end
 
+const negrevlex = neginvlex
```

Comment:
Why is this needed?

Reply:
Has been removed, I just did not understand how the `deprecate` business works.